packages/ui/src/organisms/section-pagination.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { SectionPagination } from './section-pagination';
+import { within } from '@storybook/testing-library';
+import { expect } from '@storybook/jest';
+
+const meta = {
+  component: SectionPagination,
+  title: 'UI/SectionPagination',
+} satisfies Meta<typeof SectionPagination>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const Default: Story = {
+  args: {
+    currentPage: 1,
+    totalPages: 10,
+    onPageChange: () => {
+      console.log('Page changed');
+    },
```

Comment:
Please remove. It's automatically handled by storybook when you use `on*` for your signals